_resources/port1.0/group/rust_build-1.0.tcl

```diff
@@ -0,0 +1,385 @@
+# -*- coding: utf-8; mode: tcl; c-basic-offset: 4; indent-tabs-mode: nil; tab-width: 4; truncate-lines: t -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+#
+# This PortGroup supports the Rust language ecosystem
```

Comment:
Something that I just realized when re-reviewing: this feels obvious right now, but would it not be better to say something like:

```tcl
# This PortGroup supports building core functionality of the Rust language ecosystem:
# the compiler (rustc), the standard library, and the package / build system (cargo).
# Normal Rust ports do not need to use this, but this is required for cargo, rust, and
# rust-bootstrap.
```

Reply:
I added some additional comments.